avd_docs/aws/rds/AVD-AWS-0177/docs.md

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
This documentation is generated by running `make docs`. Remember to `git add` the resultant file(s).